### ISSUE TITLE
mx.ui.action does not work as expected in popups

### ISSUE THREAD

**Issue by Andries-Smit** [mendix/generator-mendix#28]
Sharing some information:
https://github.com/mendix/generator-mendix/blob/master/generators/app/templates/WidgetBoilerplate/src/WidgetName/widget/WidgetName.js.ejs#L76

Found a limitation on the https://apidocs.mendix.com/6/client/mx.ui.html#.action
Though documentation states that `origin: this.mxform` is not needed, it is not completely true. It is not working for full pages but does not work for popups. So this call action will close the page but not a popup. : (

So I think we need to reintroduce the origin again.

Cheers.

**Comment by JelteMX** [comment]
I'll think we need to bring this up @ R&D, because adding `origin: this.mxform` results in a MX7 deprecation warning. That beings said, we can safely reintroduce it, will be deprecated in MX8 if I'm correct.

**Comment by Andries-Smit** [comment]
The deprecated variant of Mendix 5 was named `caller` not aware about the origin...

**Comment by JelteMX** [comment]
Yeah my comment was removed because I spotted my own error ;-). We'll reintroduce ``origin`` I guess, and revert back to ``mx.data.action``
